contracts/standard/arbitration/MultipleArbitrableTransaction.sol

```diff
@@ -0,0 +1,320 @@
+    
+    struct Transaction {
+        address seller;
+        address buyer;
+        uint256 amount;
+        uint256 timeout; // Time in seconds a party can take before being considered unresponding and lose the dispute.
+        uint disputeId;
+        bool disputed;
+        Arbitrator arbitrator;
+        bytes arbitratorExtraData;
+        uint sellerFee; // Total fees paid by the seller.
+        uint buyerFee; // Total fees paid by the buyer.
+        uint lastInteraction; // Last interaction for the dispute procedure.
+        Status status;
+    }
+
+    Transaction[] transactions;
+
+    mapping (bytes32 => uint) disputeTxMap;
+    
+    /** @dev Constructor.
+     */
+    function MultipleArbitrableTransaction() public {
+    }
+
+    /** @dev To be raised when a dispute is created. The main purpose of this event is to let the arbitrator know the meaning ruling IDs.
+     *  @param _transactionId The index of the transaction in dispute.
+     *  @param _arbitrator The arbitrator of the contract.
+     *  @param _disputeID ID of the dispute in the Arbitrator contract.
+     *  @param _rulingOptions Map ruling IDs to short description of the ruling in a CSV format using ";" as a delimiter. Note that ruling IDs start a 1. For example "Send funds to buyer;Send funds to seller", means that ruling 1 will make the contract send funds to the buyer and 2 to the seller.
+     */
+    event Dispute(uint indexed _transactionId, Arbitrator indexed _arbitrator, uint indexed _disputeID, string _rulingOptions);
+
+    /** @dev To be raised when a ruling is given.
+     *  @param _transactionId The index of the transaction in dispute.
+     *  @param _arbitrator The arbitrator giving the ruling.
+     *  @param _disputeID ID of the dispute in the Arbitrator contract.
+     *  @param _ruling The ruling which was given.
+     */
+    event Ruling(uint indexed _transactionId, Arbitrator indexed _arbitrator, uint indexed _disputeID, uint _ruling);
+    
+    /** @dev To be raised when evidence are submitted. Should point to the ressource (evidences are not to be stored on chain due to gas considerations).
+     *  @param _transactionId The index of the transaction.
+     *  @param _arbitrator The arbitrator of the contract.
+     *  @param _disputeID ID of the dispute in the Arbitrator contract.
+     *  @param _party The address of the party submiting the evidence. Note that 0 is kept for evidences not submitted by any party.
+     *  @param _evidence A link to evidence or if it is short the evidence itself. Can be web link ("http://X"), IPFS ("ipfs:/X") or another storing service (using the URI, see https://en.wikipedia.org/wiki/Uniform_Resource_Identifier ). One usecase of short evidence is to include the hash of the plain English contract.
+     */
+    event Evidence(uint indexed _transactionId, Arbitrator indexed _arbitrator, uint indexed _disputeID, address _party, string _evidence);
```

Comment:
`Evidence` is defined in `Arbitrable` https://github.com/kleros/kleros-interaction/blob/master/contracts/standard/arbitration/Arbitrable.sol#L45 
To keep consistency, it's better to use the same evidence event (without the `_transactionID` which can be recovered form `_party` and `_disputeID`).

Reply:
No, but `_disputeID` and `_arbitrator` makes it unique.
https://github.com/ethereum/EIPs/issues/792 : It must track the disputes by their (arbitrator,disputeID) unique key

Reply:
You are absolutely right @clesaege . I have made the requested changes now.